content/en/continuous_testing/settings/_index.md

```diff
@@ -25,9 +25,9 @@ further_reading:
 
 ## Overview
 
-You can access Continuous Testing settings on the [Synthetic Monitoring & Continuous Testing Settings page][1].
+You can access Continuous Testing settings on the [Synthetic Monitoring & Testing Settings page][1].
 
-{{< img src="continuous_testing/continuous_testing_settings_default.png" alt="Default settings for Continuous Testing" style="width:100%;">}}
+{{< img src="continuous_testing/settings/parallelization.png" alt="Set parallelization for your Continuous Testing tests on the Settings page" style="width:100%;">}}
```

Comment:
```suggestion
{{< img src="continuous_testing/settings/parallelization.png" alt="Set parallelization for your Continuous Testing tests on the Settings page" style="width:100%;">}}
```
same nit here with this image, can you re-crop it to remove the white space and gray bar on the right ?